packages/gatsby-theme-vtex/src/templates/product.tsx

```diff
@@ -81,7 +79,9 @@ const Page: FC<ProductPageProps> = (props) => {
         isPrerendered={staticPath}
         fallback={<AboveTheFoldPreview />}
       >
-        <ErrorBoundary fallback={<div>Error !!</div>}>
+        <ErrorBoundary
+          fallback={(error) => <DefaultErrorHandler error={error} />}
```

Comment:
Can't you just ?

```suggestion
          fallback={DefaultErrorHandler}
```

this way we avoid creating an unecessary function